packages/sources/svelte/src/components/overlays/VtmnAlert/VtmnAlertItem.svelte

```diff
@@ -50,7 +50,7 @@
   );
 </script>
 
-<div class={componentClass} role="alert" tabindex="-1" {...$$restProps}>
+<div class={componentClass} role="alert" aria-live="assertive" tabindex="-1" {...$$restProps}>
```

Comment:
I think that the `role="alert"` is enough here for assistive technologies.

I quote the WCAG : "_This [the aria-live="assertive" attribute] does not have to be declared in the code because it is implicit in the alert role._"
(https://www.w3.org/WAI/ARIA/apg/example-index/alert/alert.html)

What do you think ? May I have your opinion on this as well @gaspard-mathon ?

Thanks :)